framework/OutStreams/OutStreamPlot.py

```diff
@@ -1772,12 +1772,54 @@ def addOutput(self):
                   for k, col in zip(range(int(clusterDict[pltindex]['noClusters'])), colors):
                     myMembers = self.clusterValues[pltindex][1][0] == k
                     self.actPlot = plt.scatter(clusterDict[pltindex]['clusterValues'][myMembers, 0], clusterDict[pltindex]['clusterValues'][myMembers, 1] , color = col, **dataMiningPlotOptions)
+
+                  ## Handle all of the outlying data
+                  myMembers = self.clusterValues[pltindex][1][0] == -1
+                  ## resize the points
+                  dataMiningPlotOptions['s'] /= 2
+                  ## and hollow out their markers
+                  if 'facecolors' in dataMiningPlotOptions:
+                    faceColors = dataMiningPlotOptions['facecolors']
+                  else:
+                    faceColors = None
+                  dataMiningPlotOptions['facecolors'] = 'none'
+
+                  self.actPlot = plt.scatter(clusterDict[pltindex]['clusterValues'][myMembers, 0], clusterDict[pltindex]['clusterValues'][myMembers, 1] , color = '#000000', **dataMiningPlotOptions)
+
+                  ## Restore the plot options to their original values
```

Comment:
This restores the point size to its initial value so any remaining plot calls use the user-specified or default value not the reduced size points.